src/katsdpcontroller/generator.py

```diff
@@ -1199,7 +1202,7 @@ def get_baseline_index(a1, a2):
                     batches_per_chunk * stream.n_chans_per_substream * stream.spectra_per_heap
                 )
                 beam_size = elements * COMPLEX
-                rand_state_size = elements * 48  # 48 is sizeof(curandStateXORWOW_t)
+                rand_state_size = elements * 24  # 24 is sizeof(randState_t)
```

Comment:
Is this to accommodate for the increase in GPU memory usage? (Line 1330)

Doesn't necessarily seem related to NGC-1457.

Reply:
Apologies, I see the second commit corresponds to this, and the commit message explains it.

Reply:
Indeed, I probably ought to have put it in a separate PR.